### PR TITLE
Show how pagination can be done using meta tag from controller in JSON adapter

### PR DESCRIPTION
**Pull request by vasilakisfil**

### CODE REVIEW

File: docs/howto/add_pagination_links.md [hunk 1]
```diff
@@ -110,6 +110,26 @@ ex.
 }
 ```
 
+You can also achieve the same result if you have a helper method that adds the pagination info in the meta tag. For instance, in your action specify a custom serializer.
+
+```ruby
+render json: @posts, each_serializer: PostPreviewSerializer, meta: meta_attributes(@post)
```
Comment (NullVoxPopuli) [inline docs/howto/add_pagination_links.md]: @beauby does this look correct? I haven't used this feature yet

File: docs/howto/add_pagination_links.md [hunk 2]
```diff
@@ -110,6 +110,26 @@ ex.
...
+#expects pagination!
+def meta_attributes(resource, extra_meta = {})
+  {
+    current_page: resource.current_page,
+    next_page: resource.next_page,
+    prev_page: resource.prev_page,
+    total_pages: resource.total_pages,
+    total_count: resource.total_count
+  }.merge(extra_meta)
+end
```
Comment (bf4) [inline docs/howto/add_pagination_links.md]: would you mind adding a test for this example? looks good

**Comment by bf4** [comment]
@vasilakisfil are we still working on this?

**Comment by vasilakisfil** [comment]
Oops I forgot. You will have it either today or tomorrow.

**Comment by bf4** [comment]
Yay!

B mobile phone

**Comment by NullVoxPopuli** [comment]
is this still wanted in the docs?

**Comment by bf4** [comment]
@nullvoxpopuli 100x yes unless It's already there (on phone)

**Comment by NullVoxPopuli** [comment]
@bf4 you good with merging it?

**Comment by bf4** [comment]
Sure, and please add issue to add tests

**Pull request merged**